### DISCUSSION TITLE
Add Clublog confirmation column to the "Overview" log view.

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#722] [Ideas]
The Clublog confirmation column is now only shown in the advanced log view. It would be nice to see this column in the overview log view and in the QSO search results.

**Comment by AndreasK79** [comment]
Hi. This is already implemented, but it is connected to the default QSL-methods under default values in the account settings.

**Reply by R1BLH** [reply]
Thank you very much! I didn't know that (that setting wasn't there before, and you can't manually check everything every time you update). :)
